packages/reaction-collections/server/publications/products.js

```diff
@@ -7,12 +7,31 @@
 Meteor.publish("Products", function (productScrollLimit, productFilters) {
   check(productScrollLimit, Match.OneOf(null, undefined, Number));
```

Comment:
I have a feeling that we can't use `audit-arguments-check` and `SS.validator()` within one app. [here](https://forums.meteor.com/t/using-simpleschema-for-argument-check-in-publications-and-audit-arguments-check/16251)